src/models/role.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum RoleKey {
+  Admin = 'Admin',
```

Comment:
suggestion-- should not be placed here .. can be in a separate folder

Reply:
What should be the folder name and location? Any convention that we follow?